### DISCUSSION TITLE
Erro ao recriar Status Screen

### DISCUSSION THREAD

**Discussion by wjrcode** [mercadopago/sdk-js#211] [Q&A] [answered]
Preciso recriar o Status Screen para exibir que o pix foi recebido com sucesso, porém quando eu tento recriar o componente ele fica carregando e retorna um erro.

https://codesandbox.io/p/sandbox/vgff3p

![image_2024_09_27T20_04_11_902Z-1 (1)](https://github.com/user-attachments/assets/3d1f0201-c69e-421b-abca-592dc1df4444)

**Comment by icaldana** [comment, marked as answer]
Olá @wjrcode!

Encontramos um bug que acontece ao tentar criar um novo Status Screen após a execução do método `unmount`. Estamos trabalhando no fix e traremos novidades em breve.

**Reply by wjrcode** [reply]
Ok, obrigado! Consigo acompanhar isso?

**Reply by icaldana** [reply]
Bom dia! 

A correção já está disponível, pode validar pelo [sandbox](https://codesandbox.io/p/sandbox/vgff3p) (talvez seja necessário limpar o cache em seu navegador).

Muito obrigado!

**Reply by wjrcode** [reply]
Tudo certo, obrigado!
